gmd-sdk/nifi-data-processors/src/main/scala/com/deciphernow/greymatter/data/nifi/properties/GetOidForPathProperties.scala

```diff
@@ -13,42 +13,44 @@ import org.apache.nifi.processor.util.StandardValidators
 
 trait GetOidForPathProperties extends CommonProperties with ErrorHandling {
 
-  protected lazy val userfieldObjectPolicyProperty = buildProperty("Userfield Folder Object Policy", "When provided, this is an override object policy to be assigned to the created userfield folder if the folder does not yet exist.").build()
+  protected lazy val userfieldObjectPolicyProperty = buildPropertyWithValidators(List(StandardValidators.ATTRIBUTE_EXPRESSION_LANGUAGE_VALIDATOR),"Userfield Folder Object Policy", "When provided, this is an override object policy to be assigned to the created userfield folder if the folder does not yet exist.", ExpressionLanguageScope.FLOWFILE_ATTRIBUTES).defaultValue("${gmdata.userfielfolderdobjectpolicy}").build()
```

Comment:
The default here appears to be spelled wrong.  I think this should be 
`${gmdata.userfieldfolderobjectpolicy}`